iosApp/Podfile.lock

```diff
@@ -22,4 +22,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 32413a7fdb16d40c9f57cf8da0c74d4507ff5f90
 
-COCOAPODS: 1.15.2
+COCOAPODS: 1.16.2
```

Comment:
I'm sneaking this in here since we may as well sneak it in somewhere - as discussed at some point, it's not just me who's been manually keeping this change out of my Git commits for a while.